.hooks/plugin-warning-logger.js

```diff
@@ -16,7 +16,9 @@ module.exports = {
         }
       },
       wrapScriptExecution(executor, project, locator, scriptName) {
```

Comment:
This is what shows the
```
╔══════════════════════════════════════════════════════════════════════════════╗
║ Be sure to copy a config/xxx.json to build/config.json if you're planning to ║
║ deploy this build.                                                           ║
╚══════════════════════════════════════════════════════════════════════════════╝
```
notice after running a build. We only want to show that on app builds, not package builds.

Reply:
Yes, it's done in CI. 

https://github.com/DataBiosphere/terra-ui/blob/ecc6bf1e38d5b92975f5178860a4ca26b470a5e4/.circleci/config.yml#L117
https://github.com/DataBiosphere/saturn-ui-prod-deploy/blob/44b4565a4dc118482be63e12d6497ab62271fcb0/.circleci/config.yml#L76

Yeah, we could probably get rid of this altogether.

Reply:
That is exactly why it is there. If I had the time, I'd instead move this warning into an error message that shows up when the UI attempts to pull that file and finds it absent. This warning was added specifically because the mechanism for resolving that error isn't particularly obvious.